lib/vcloud/converter.rb

```diff
@@ -1,8 +1,33 @@
 require "vcloud/converter/version"
+require "yaml"
+require "erb"
 
 module Vcloud
   module Converter
     def self.convert(conversion_type, file_path)
+      if conversion_type == "vcloud-net_launcher"
+        convert_vcloud_net_launcher(file_path)
+      end
+    end
+
+    def self.convert_vcloud_net_launcher(file_path)
+      vcloud_config = YAML.load_file(file_path)
```

Comment:
Finally got around to doing this (and catching non-existent files people try to convert).